projects/etos_suite_runner/pyproject.toml

```diff
@@ -18,8 +18,8 @@ requires-python = ">=3.9"
 dependencies = [
     "packageurl-python~=0.11",
     "cryptography>=42.0.4,<43.0.0",
-    "etos_lib==4.5.0",
-    "etos_environment_provider==5.3.0",
+    "etos_lib==5.1.2",
+    "etos_environment_provider==5.3.1",
```

Comment:
The new version of the environment-provider is assumed to be 5.3.1, but it could be subject to change. This is also why CI actions fail.